easybuild/easyblocks/c/clang.py

```diff
@@ -479,6 +479,17 @@ def sanity_check_step(self):
         if LooseVersion(self.version) >= LooseVersion('3.8'):
             custom_paths['files'].extend(["lib/libomp.%s" % shlib_ext, "lib/clang/%s/include/omp.h" % self.version])
 
+        if 'NVPTX' in self.cfg['build_targets']:
+            arch = get_cpu_architecture()
+            if arch == POWER:
+                # The 'libomptarget.rtl' on POWER systems is identified using
+                # the following arch
+                arch = 'ppc64'
```

Comment:
@nordmoen Let's also cover `aarch64` here, which has `libomptarget.rtl.aarch64.so`:

```python
elif arch == AARCH64:
    arch = 'aarch64'
```

Or equivalenty:

```python
else:
    arch = arch.lower()
```

Reply:
I had a feeling it would be wrong on `AArch64`. I have change the code to do explicit comparison against the different architectures so that we are not dependent on the constants.